component/authenticator/src/webapp/voiceMobile.jsp

```diff
@@ -0,0 +1,203 @@
+<%--
```

Comment:
```suggestion
<%--
  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  ~
  ~ WSO2 Inc. licenses this file to you under the Apache License,
  ~ Version 2.0 (the "License"); you may not use this file except
  ~ in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~ http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing,
  ~ software distributed under the License is distributed on an
  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
  ~ KIND, either express or implied.  See the License for the
  ~ specific language governing permissions and limitations
  ~ under the License.
  --%>
```

Add the correct license header with the appropriate year. You can refer to it here. https://github.com/wso2/identity-apps/blob/35af68035b1235dd420116e691de8db02bdb1292/identity-apps-core/apps/x509-certificate-authentication-portal/src/main/webapp/tenant-resolve.jsp#L1-L17